src/sequences/SequenceEntity.js

```diff
@@ -18,19 +18,33 @@ export class SequenceEntity extends Entity {
 		return this._entity && this._entity.properties && this._entity.properties.title;
 	}
 
+	sequenceViewerApplicationHref() {
+		const sequenceViewerRel = 'https://sequences.api.brightspace.com/rels/sequence-viewer-application';
+		return this._entity && this._entity.hasLinkByRel(sequenceViewerRel) &&
+		this._entity.getLinkByRel(sequenceViewerRel).href;
+	}
+
 	onSubSequencesChange(onChange) {
 		const subSequences = this._subSequences();
 
-		subSequences.forEach((entity) => {
-			entity && this._subEntity(SequenceEntity, entity, onChange);
+		subSequences.forEach((entity, index) => {
+			const onChangeWithIndex = (subSequence) => {
```

Comment:
this feels like a not hypermedia way to do this.

Do you have a sample json?  I'd expect the entities to each have links to next and previous entities, which you could then use to create your sequence (simliar to a linked list i guess?)

Reply:
I do, I'll send it. I don't see the next/previous at this level.